Godeps/_workspace/src/k8s.io/kubernetes/pkg/storage/etcd/etcd_helper.go

```diff
@@ -313,6 +313,20 @@ func (h *etcdHelper) Delete(ctx context.Context, key string, out runtime.Object)
 }
 
 // Implements storage.Interface.
+func (h *etcdHelper) DeleteDir(ctx context.Context, directory string) error {
```

Comment:
I think this should take the following:

`DeleteDir(ctx context.Context, key string, typeName string)`